app/src/components/steps/add-inventory-details-step.tsx

```diff
@@ -0,0 +1,349 @@
+                  shadow="1dp"
+                  fontSize="body.lg"
+                  fontStyle="normal"
+                  letterSpacing="wide"
+                  _placeholder={{ color: "content.tertiary" }}
+                  py="16px"
+                  px={0}
+                  {...register("year", {
+                    required: t("inventory-year-required"),
+                  })}
+                >
+                  {years.map((year: number, i: number) => (
+                    <option value={year} key={i}>
+                      {year}
+                    </option>
+                  ))}
+                </Select>
+                <InputRightElement>
+                  {!!year && (
+                    <CheckIcon
+                      color="semantic.success"
+                      boxSize={4}
+                      mt={2}
+                      mr={10}
+                    />
+                  )}
+                </InputRightElement>
+              </InputGroup>
+              <FormErrorMessage gap="6px" m={0}>
+                <WarningIcon h="16px" w="16px" />
+                <Text
+                  fontSize="body.md"
+                  color="content.tertiary"
+                  fontStyle="normal"
+                >
+                  {errors.year && errors.year.message}
+                </Text>
+              </FormErrorMessage>
+            </FormControl>
+          </Box>
+        </Box>
+      </Box>
+      {/* Inventory Goal */}
+      <Box
+        w="full"
+        py="36px"
+        borderBottomWidth="2px"
+        borderColor="border.overlay"
+      >
+        <Box
+          display="flex"
+          w="full"
+          alignItems="center"
+          justifyContent="space-between"
+        >
+          <Box display="flex" flexDir="column" gap="16px">
+            <Text
+              fontFamily="heading"
+              fontSize="title.md"
+              fontStyle="normal"
+              fontWeight="bold"
+              lineHeight="24px"
+            >
+              {t("inventory-goal")}
+            </Text>
+            <Text
+              fontSize="title.md"
+              fontStyle="normal"
+              lineHeight="24px"
+              letterSpacing="wide"
+              color="content.tertiary"
+            >
+              <Trans i18nKey="inventory-goal-description" t={t}>
+                Want to learn more about these inventory formats?{" "}
+                <Link
+                  href="/"
+                  fontFamily="heading"
+                  fontWeight="bold"
+                  color="content.link"
+                  textDecorationLine="underline"
+                >
+                  Learn more
+                </Link>{" "}
+                about the GPC Framework.
+              </Trans>
+            </Text>
+          </Box>
+          <Box>
+            {/* TODO:
+              only enable basic by default and disable basic+ until we have the feature
+              */}
```

Comment:
I think this has been implemented now, so it's okay to enable BASIC+ now ✨
And to remove the TODO comment.